.env.example

```diff
@@ -1,7 +1,4 @@
 CLIENT_ID=
 CLIENT_SECRET=
 
-RENDER_IMAGE=hugo_render
-VOLUME_DATA=/home//wikigdrive
-VOLUME_PREVIEW=/home//wikigdrive-preview
-DOMAIN=localhost
+DOMAIN=http://localhost:3000
```

Comment:
do we no longer need `RENDER_IMAGE` , `VOLUME_DATA` and `VOLUME_PREVIEW`?  
maybe we should consider adding here placeholders for variables related to JWT and zipkin URL?

Reply:
In https://github.com/mieweb/wikiGDrive/blob/develop/website/docs/developer-guide.md#running-locally-with-docker we specify those env variables in `docker run` command

I we use both `.env` file and `-e RENDER_IMAGE...` it's not clear which env would be used by wikigdrive.